std/container/rbtree.d

```diff
@@ -1833,6 +1912,36 @@ pure unittest
     assert(equal(rbt[], [4, 3, 2, 1, 0]));
 }
 
+// convenience wrapper range construction
+pure unittest
+{
+    import std.algorithm : equal;
+    import std.range : iota;
+    auto rbt = redBlackTree!"a > b"(iota(5));
+    assert(equal(rbt[], [4, 3, 2, 1, 0]));
+
+    auto rbt2 = redBlackTree!"a > b"(["a", "b"]);
```

Comment:
Right .. I left the array based tests and added new ones for ranges with `iota` and `chain`.